io/disk/iozone.py

```diff
@@ -0,0 +1,649 @@
+import os
+import re
+import json
+import logging
+import math
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils import distro
+from avocado.utils.software_manager import SoftwareManager
+
+
+_LABELS = ['file_size', 'record_size', 'write', 'rewrite', 'read', 'reread',
+           'randread', 'randwrite', 'bkwdread', 'recordrewrite', 'strideread',
+           'fwrite', 'frewrite', 'fread', 'freread']
+
+
+def unique(list):
```

Comment:
It's not really nice to replace builtins with params, please use something else than list. (eg. items, or inp_list)